composer.json

```diff
@@ -54,22 +54,28 @@
         "drupal/field_group": "3.1",
         "drupal/field_group_link": "3.0-rc2",
         "drupal/image_widget_crop": "2.3",
+        "drupal/link_class": "2.0.0",
         "drupal/media_library_form_element": "2.0.3",
         "drupal/media_library_theme_reset": "1.0",
         "drupal/metatag": "1.15",
         "drupal/migrate_plus": "5.1",
         "drupal/migrate_tools": "5.0",
         "drupal/paragraphs": "1.12",
         "drupal/pathauto": "1.8",
+        "drupal/rabbit_hole": "1.0-beta10",
         "drupal/redirect": "1.6",
+        "drupal/slick": "2.2",
```

Comment:
`drupal/slick` requires `drupal/blazy` in its own composer.json file:
```
  "require": {
    "drupal/blazy": "~2.0"
  }
```

It might be good to add `drupal/blazy` as an explicit dependency to az_quickstart so we can pin to a specific version for more deterministic builds...

Reply:
Update: decided not to do this––allow the modules to track their own dependencies.